### PR TITLE
activated angularCompiler Strict Mode

### PR DESCRIPTION
**Pull request by fabianpage**
`app-individual-subscription` doesn't exists anymore.

### CODE REVIEW

**Review by rawyler — APPROVED** [review]

**Review by pgoellnitz — APPROVED** [review]
Ok for me, however the alignment of the buttons should not change.

File: src/app/core/not-found.component.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
           | translate
       }}</mat-card-subtitle>
     </mat-card-header>
-    <mat-card-actions align="center">
```
[the remark this answers is not in the document]
Comment (fabianpage) [reply src/app/core/not-found.component.html:10]: unfortunately align can only have the values 'start' or 'end'.

CleanShot 2022-02-16 at 16 31 36@2x

[the remark this answers is not in the document]
Comment (fabianpage) [reply src/app/core/not-found.component.html:10]: hm, but on app.phaenonet.ch it does center the action. I will have a look.

[the remark this answers is not in the document]
Comment (pgoellnitz) [reply src/app/core/not-found.component.html:10]: Just checked on a channel. Actually it's still centered with the change.

File: src/app/login/logged-out.component.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
       <mat-card-title>{{ 'Bis bald' | translate }}</mat-card-title>
       <mat-card-subtitle>{{ 'Sie haben sich erfolgreich ausgeloggt' | translate }}</mat-card-subtitle>
     </mat-card-header>
-    <mat-card-actions align="center">
```
[the remark this answers is not in the document]
Comment (pgoellnitz) [reply src/app/login/logged-out.component.html:7]: checked, no change in alignment.

File: src/app/core/not-found.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (pgoellnitz) [reply src/app/core/not-found.component.html:10]: any possibility to keep the alignment and ignore the issue? Like for linting issues.

[the remark this answers is not in the document]
Comment (fabianpage) [reply src/app/core/not-found.component.html:10]: I will overwrite it with css. It's not the nicest solution but the only one i found.

[the remark this answers is not in the document]
Comment (fabianpage) [reply src/app/core/not-found.component.html:10]: See the fix in not-found.component.html, logged-out.component.html and _overwrite-mat.scss.

File: src/app/login/logged-out.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (fabianpage) [reply src/app/login/logged-out.component.html:7]: See the fix in not-found.component.html, logged-out.component.html and _overwrite-mat.scss.

**Pull request merged**